tests/FSharp.Stats.Tests/Formatting.fs

```diff
@@ -0,0 +1,86 @@
+﻿module FormattingTests
```

Comment:
Are nans and infinities printed properly?

Reply:
I added tests for both individual formatting and a matrix containing mixed values of all cases.